### ISSUE TITLE
Time for a v0.25 release?

### ISSUE THREAD

**Issue by petergeoghegan** [EUA/wxHexEditor#198]
It would be nice if you could create a new release, which would presumably become available via standard package managers before too long. There have been some useful bug fixes and enhancements since v0.24, and it's unfortunate that users are currently required to build from source to get those.

Thanks!

**Comment by EUA** [comment]
Yeah. Time has come. But I think I need to make bug fixing first.

**Comment by pcjc2** [comment]
Just adding myself as a subscriber here, as a new binary release for Win32 would be appeciated.
